src/pages/nickname.html

```diff
@@ -12,7 +12,15 @@
     <main class="container">
         <h3 class="container__title">Enter your nickname:</h3>
         <input class="nickname-input" type="text" name="nickname" value="" placeholder="nickname">
-        <button class="btn">Enter!</button>
+        <a href="question.html" style="text-decoration: none;color:inherit">
+        <button class="btn">Enter!</button></a>
     </main>
+    <script defer type="module">
+        import {saveToLocalStorage, loadFromLocalStorage} from "../js/localstorage.js"
+
+        const button = document.querySelector("button");
+        button.addEventListener("click", function () {
+        saveToLocalStorage(document.querySelector('input').value, "enterName");});
```

Comment:
You've created dedicated functions to write objects to localStorage. Use them instead. We will have to change the key name in every place if we use it like there if we would like to change it.

Reply:
Done.